src/common/exception_format_value.cpp

```diff
@@ -24,6 +25,9 @@ ExceptionFormatValue::ExceptionFormatValue(uhugeint_t uhuge_val)
 ExceptionFormatValue::ExceptionFormatValue(string str_val)
     : type(ExceptionFormatValueType::FORMAT_VALUE_TYPE_STRING), str_val(std::move(str_val)) {
 }
+ExceptionFormatValue::ExceptionFormatValue(String str_val)
+    : type(ExceptionFormatValueType::FORMAT_VALUE_TYPE_STRING), str_val(std::move(str_val.ToStdString())) {
+}
```

Comment:
Can we remove `ExceptionFormatValue(string str_val)`? It looks like `String` allows implicit conversion with its owning (`String(std::string str)`) constructor? Same for `CreateFormatValue`.